.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rust-toolchain: [nightly-2024-05-02, nightly-2024-08-24]
+        rust-toolchain: [nightly, nightly-2024-05-02]
```

Comment:
Why should we change this?

Reply:
I think that by using the latest nightly toolchain we can keep up with the newest modification brought by rust community

Reply:
errors have been bypassed by `continue-on-error`

Reply:
It's weird as `naked_asm!` in newer versions has already messed thing up once and we've decided to stick to older versions for a while until edition 2024 is released.

Reply:
If we use `continue-on-error`, it means the "newest modification brought by rust community" was not used by us at all.

Reply:
> If we use `continue-on-error`, it means the "newest modification brought by rust community" was not used by us at al

but we can catch up with which part of our code is compatible with the  "newest modification brought by rust community"

Reply:
> > If we use `continue-on-error`, it means the "newest modification brought by rust community" was not used by us at al
> 
> but we can catch up with which part of our code is compatible with the "newest modification brought by rust community"

OK, this is a good enough reason.